internal/config/cgroupmanager/systemdmanager.go

```diff
@@ -0,0 +1,148 @@
+// +build linux
+
+package cgroupmanager
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/cri-o/cri-o/utils"
+	"github.com/opencontainers/runc/libcontainer/cgroups/systemd"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const defaultSystemdParent = "system.slice"
+
+// SystemdManager is the parent type of SystemdV{1,2}Manager.
+// it defines all of the common functionality between V1 and V2
+type SystemdManager struct{}
+
+// Name returns the name of the cgroup manager (systemd)
+func (*SystemdManager) Name() string {
+	return systemdCgroupManager
+}
+
+// IsSystemd returns that it is a systemd cgroup manager
+func (*SystemdManager) IsSystemd() bool {
+	return true
+}
+
+// GetContainerCgroupPath takes arguments sandbox parent cgroup and container ID and returns
+// the cgroup path for that containerID. If parentCgroup is empty, it
+// uses the default parent system.slice
+func (*SystemdManager) GetContainerCgroupPath(sbParent, containerID string) string {
+	parent := defaultSystemdParent
+	if sbParent != "" {
+		parent = sbParent
+	}
+	return parent + ":" + scopePrefix + ":" + containerID
+}
+
+// MoveConmonToCgroup takes the container ID, cgroup parent, conmon's cgroup (from the config) and conmon's PID
+// It attempts to move conmon to the correct cgroup.
+func (*SystemdManager) MoveConmonToCgroup(cid, cgroupParent, conmonCgroup string, pid int) (string, error) {
```

Comment:
I think it deserves to mention what the return value is, same as in other places.

Reply:
added a comment :+1: